lib/src/client/call.dart

```diff
@@ -0,0 +1,285 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:http2/transport.dart';
+
+import '../shared/status.dart';
+import '../shared/streams.dart';
+
+import 'client.dart';
```

Comment:
This dependency is part of a cycle. You may be able to break it by moving `ClientMethod` here or to its own `client/method.dart` file.